examples/ts-ecr-dockerfile/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "fargate",
```

Comment:
Copy-paste name? Let's rename it and bump nodejs to 18+.

I've also found that I jsut had to change a number of other examples to move awsx from `peerDependencies` to actual `dependencies`, other yarn/npm won't install it.